doc/modules/ROOT/pages/migration-alpha-cp-to-cpv2/index.adoc

```diff
@@ -1,6 +1,6 @@
 [appendix]
 [[appendix-d]]
-= Migration from Alpha Cypher Aggregation to new Cypher projection
+= Migrate from Alpha Cypher to Cypher
```

Comment:
This one has to include the word "projection". Saying "Cypher" without that qualifier is misleading.

The "aggregation" qualifier is maybe optional, but it is useful to reference the initial name we used for this concept. I don't know how well-spread that name is, but the operation is well described on this page as "Cypher Aggregation" so maybe it is good to retain it.

We could skip "Alpha" IMO. It doesn't add too much here, and is obvious from the name of the function, which includes the namespace.

I'm not bound on "new", but it is good with some kind of differentiating term to make it clear that the Cypher Projections we've had for a long time, now deprecated, are not the target here.

```suggestion
= Migrate from Cypher Aggregation to new Cypher Projection
```

Reply:
57 chars

Reply:
Ok, the only thing is that we don't capitalize titles, but I wonder whether Aggregation and Projection are part of the name of something?

Reply:
Projection is part of the name, yes, but I would agree with you to not capitalise the title